plugin/src/test/groovy/org/gradle/testretry/testframework/JUnit4FuncTest.groovy

```diff
@@ -375,6 +375,39 @@ class JUnit4FuncTest extends AbstractFrameworkFuncTest {
         gradleVersion << GRADLE_VERSIONS_UNDER_TEST
     }
 
+    def "test that is skipped after failure with option 'failOnSkippedAfterRetry = false' is passes (gradle version #gradleVersion)"() {
```

Comment:
Please also add a test checking the default behavior that retries will fail.

Reply:
it is presented in [test above](https://github.com/gradle/test-retry-gradle-plugin/blob/99cb2792f6c0020e825a71f37a879f9526e81169/plugin/src/test/groovy/org/gradle/testretry/testframework/JUnit4FuncTest.groovy#L346)

Reply:
Ah, right, missed that. Looks good then.